sub_workflows/prepare_regions.cwl

```diff
@@ -98,25 +99,28 @@ steps:
 
   gatk_intervallisttools_concat_supplement:
     run: ../tools/gatk_intervallisttools.cwl
-    when: $(inputs.input[3] != null)
+    when: $(inputs.run_step != null)
     in:
+      run_step: supplement_vcfs
```

Comment:
Ah, ok, kind of doing like a variable assignment for clarity instead of referring to an array position, nice